src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -54,12 +54,12 @@ public function copy($originFile, $targetFile, $overwriteNewerFiles = false)
 
         if ($doCopy) {
             // https://bugs.php.net/bug.php?id=64634
-            if (false === $source = @fopen($originFile, 'r')) {
+            if (false === $source = @fopen($originFile, 'rb')) {
```

Comment:
This should be reverted no? (same for the other same changes)

Reply:
Yep, but CS bot requires the changes.

Reply:
Oh, okay :man_shrugging:

Reply:
can you revert? that's a bug of the cs bot (already reported on cs-fixer)

Reply:
Reverting =)

Reply:
Done!